exercises/pig-latin/pig_latin_test.go

```diff
@@ -1,11 +1,15 @@
-package igpay
+package piglatin
 
-import "testing"
+import (
+	"fmt"
+	"testing"
+)
 
 func TestPigLatin(t *testing.T) {
 	for _, test := range testCases {
-		if pl := PigLatin(test.input); pl != test.expected {
-			t.Fatalf("FAIL: %s\nPigLatin(%q) = %q, want %q.", test.description, test.input, pl, test.expected)
+		fmt.Printf("Test: %s\n", test.description)
```

Comment:
If test fails we will see it twice, let's write it after `if` and also we can write `PASS` instead of `Test` since we know it passed.